benchmarks/configs/pretraining_experiments.py

```diff
@@ -127,9 +127,9 @@
             )
         ),  # use spiral policy for more even object coverage during learning
     ),
-    dataset_args=PatchViewFinderMountHabitatDatasetArgs(),
-    train_dataloader_class=ED.InformedEnvironmentDataLoader,
-    train_dataloader_args=EnvironmentDataloaderPerObjectArgs(
+    env_interface_config=PatchViewFinderMountHabitatEnvironmentArgs(),
```

Comment:
same here. If it is used in the `env_interface_config` it seems like the dataclass should end in `Config` and if it is used in the `env_interface_args` it should end in `Args`. Although in the long term maybe we can think of a more simplified structure for those config fields in general (not in the scope of this PR)